src/writeData/components/WriteDataHelperBuckets.tsx

```diff
@@ -59,8 +59,8 @@ const WriteDataHelperBuckets: FC = () => {
     body = (
       <List
         backgroundColor={InfluxColors.Obsidian}
-        style={{height: '118px'}}
-        maxHeight="118px"
+        style={{height: '200px'}}
+        maxHeight="200px"
```

Comment:
fixes this: 
![image](https://user-images.githubusercontent.com/4805997/128900153-8cb397c0-1134-4751-ab56-4272eda8b9fc.png)